### PR TITLE
Remove sass mixins for prefixes

### PR DESCRIPTION
**Pull request by TrySound**
Most prefixes are not necessary in latest browsers.
user-select, animation and keyframes are stil prefixed.

Default browserslist config is `> 0.5%, last 2 versions, Firefox ESR, not dead`

### CODE REVIEW

**Comment by jakearchibald** [comment]
Do you think we need autoprefixer? There are so few cases where prefixes are needed now, they can just be done manually.

**Comment by TrySound** [comment]
Dunno user-select, animation, keyframes, min-content, max-content are still prefixed in popular browsers

**Comment by jakearchibald** [comment]
`user-select` needs a webkit prefix, but I don't think the others have needed a prefix for a long time.

- https://caniuse.com/mdn-css_properties_user-select
- https://caniuse.com/css-animation
- https://caniuse.com/mdn-css_at-rules_keyframes
- https://caniuse.com/mdn-css_properties_width_min-content
- https://caniuse.com/mdn-css_properties_width_max-content

**Comment by TrySound** [comment]
Ok, makes sense since it's development tool

**Review by jakearchibald — APPROVED** [review]
LGTM! Should make the CSS significantly smaller.

File: src/css/_utils.scss [hunk 1]
```diff
@@ -1,16 +1,7 @@
-@mixin placeholder {
-  &::-webkit-input-placeholder,
-  &::-moz-placeholder,
-  &:-ms-input-placeholder {
-    @content;
-  }
-}
-
 @mixin user-select($spec...) {
   -webkit-touch-callout: $spec;
```
Comment (XhmikosR) [inline src/css/_utils.scss:2]: @TrySound Isn't this obsolete? It could probably be removed IIRC.

Comment (TrySound) [reply src/css/_utils.scss:2]: 🤷‍♂️ Maybe

**Pull request merged**